driver/parser/check_test.go

```diff
@@ -496,28 +487,45 @@ func TestScenario_NodeGenesisIssuesAreDetected(t *testing.T) {
 		Name:     "Test",
 		Duration: 60,
 		Nodes: []Node{
-			{Genesis: Genesis{Import: &GenesisTarget{
-				Start: start,
-				Path:  "/does/exist.notg",
-			}}},
+			{Genesis: Genesis{Import: "/does/exist.notg"}},
 		},
 	}
 	if err := scenario.Check(); err == nil || !strings.Contains(err.Error(), "provided path is not a genesis file") {
 		t.Errorf("targeted file is not a genesis but issue was not detected")
 	}
+}
+
+func TestScenario_NodeGenesisExportIssuesAreDetected(t *testing.T) {
+	scenario := Scenario{
+		Name:     "Test",
+		Duration: 60,
+		Nodes: []Node{
+			{Genesis: Genesis{Export: "/file/exists.g"}},
+		},
+	}
+	if err := scenario.Check(); err == nil || !strings.Contains(err.Error(), "provided genesis file already exists") {
+		t.Errorf("genesis does not exist but issue was not detected")
+	}
 
-	*start = 70
 	scenario = Scenario{
 		Name:     "Test",
 		Duration: 60,
 		Nodes: []Node{
-			{Genesis: Genesis{Import: &GenesisTarget{
-				Start: start,
-				Path:  "/does/exist.g",
-			}}},
+			{Genesis: Genesis{Export: ""}},
 		},
 	}
-	if err := scenario.Check(); err == nil || !strings.Contains(err.Error(), "event start must be <= node end") {
-		t.Errorf("genesis import/export outside duration but issue was not detected")
+	if err := scenario.Check(); err == nil || !strings.Contains(err.Error(), "provided path to genesis file is nil") {
+		t.Errorf("targeted path was nil but issue was not detected")
+	}
+
+	scenario = Scenario{
+		Name:     "Test",
+		Duration: 60,
+		Nodes: []Node{
+			{Genesis: Genesis{Export: "/does/exist.notg"}},
+		},
+	}
+	if err := scenario.Check(); err == nil || !strings.Contains(err.Error(), "provided path is not a genesis file") {
+		t.Errorf("targeted file is not a genesis but issue was not detected")
```

Comment:
Nit: not sure if this is necessary - the user probably knows what extension they want.
No strong opinion either way.

Reply:
You have a good point. I will remove it.